reference/UiTPAS.v2.json

```diff
@@ -334,150 +352,955 @@
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          }
+        },
+        "operationId": "get-financials-organizer-periods",
+        "security": [
+          {
+            "User Acces Token (see Authentication docs)": []
+          }
+        ],
+        "description": "Retrieve report periods for an organizer"
+      }
+    },
+    "/financials/organizers/{organizerId}/reports": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "organizerId",
+          "in": "path",
+          "required": true
+        }
+      ],
+      "post": {
+        "summary": "Start a financial report export for an organizer",
+        "operationId": "post-financials-organizers-reports",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ReportStatus"
+                },
+                "examples": {
+                  "Example of started export": {
+                    "value": {
+                      "status": "started",
+                      "reportId": "232432"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          }
+        },
+        "tags": [
+          "Financial"
+        ],
+        "description": "Starts a financial report export. The result of this request is a `reportId` that can be used in the /status and /download requests."
+      }
+    },
+    "/financials/organizers/{organizerId}/report/{reportId}/status": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "organizerId",
+          "in": "path",
+          "required": true
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "reportId",
+          "in": "path",
+          "required": true
+        }
+      ],
+      "get": {
+        "summary": "Get financial report status",
+        "tags": [
+          "Financial"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ReportStatus"
+                },
+                "examples": {
+                  "Example of a started report": {
+                    "value": {
+                      "reportId": "232432",
```

Comment:
De reportId zit al mee in de url dus ik zou die hier niet mee in steken. `status` lijkt me voldoende, en eventueel een link naar de download (zie andere comment)